app.py

```diff
@@ -0,0 +1,213 @@
+    container_name="agora-api-docs",
+    container_location=f"ghcr.io/sage-bionetworks/agora-api-docs:{agora_version}",
+    container_port=8010,
+    container_memory=200,
+    container_env_vars={"PORT": "8010"},
+)
+api_docs_stack = ServiceStack(
+    scope=cdk_app,
+    construct_id=f"{stack_name_prefix}-api-docs",
+    vpc=network_stack.vpc,
+    cluster=ecs_stack.cluster,
+    props=api_docs_props,
+)
+
+mongo_props = ServiceProps(
+    container_name="agora-mongo",
+    container_location=f"ghcr.io/sage-bionetworks/agora-mongo:{agora_version}",
+    container_port=27017,
+    container_memory=500,
+    container_env_vars={
+        "MONGO_INITDB_ROOT_USERNAME": "root",
+        "MONGO_INITDB_ROOT_PASSWORD": "changeme",
+        "MONGO_INITDB_DATABASE": "agora",
+    },
+    container_volumes=[
+        ContainerVolume(
+            path="/data/db",
+            size=30,
+        )
+    ],
+)
+mongo_stack = ServiceStack(
+    scope=cdk_app,
+    construct_id=f"{stack_name_prefix}-mongo",
+    vpc=network_stack.vpc,
+    cluster=ecs_stack.cluster,
+    props=mongo_props,
+)
+
+# It is probably not appropriate host this container in ECS
+# data_props = ServiceProps(
+#     container_name="agora-data",
+#     container_location=f"ghcr.io/sage-bionetworks/agora-data:{agora_version}",
+#     container_port=9999,  # Not used
+#     container_memory=2048,
+# )
+# data_stack = ServiceStack(
+#     scope=cdk_app,
+#     construct_id=f"{stack_name_prefix}-data",
+#     vpc=network_stack.vpc,
+#     cluster=ecs_stack.cluster,
+#     props=data_props,
+#     container_env_vars={
+#         "DB_USER": "root",
+#         "DB_PASS": "changeme",
+#         "DB_NAME": "agora",
+#         "DB_PORT": "27017",
+#         "DB_HOST": "agora-mongo",
+#         "DATA_FILE": "syn13363290",
+#         "DATA_VERSION": "68",
+#         "TEAM_IMAGES_ID": "syn12861877",
+#         "SYNAPSE_AUTH_TOKEN": "agora-service-user-pat-here",
+#     },
+# )
+# data_stack.add_dependency(mongo_stack)
+
+api_props = ServiceProps(
+    container_name="agora-api",
+    container_location=f"ghcr.io/sage-bionetworks/agora-api:{agora_version}",
+    container_port=3333,
+    container_memory=1024,
+    container_env_vars={
+        "MONGODB_URI": "mongodb://root:changeme@agora-mongo:27017/agora?authSource=admin",
+        "NODE_ENV": "development",
+    },
+)
+api_stack = ServiceStack(
+    scope=cdk_app,
+    construct_id=f"{stack_name_prefix}-api",
+    vpc=network_stack.vpc,
+    cluster=ecs_stack.cluster,
+    props=api_props,
+)
+api_stack.add_dependency(mongo_stack)
+
+app_props = ServiceProps(
+    container_name="agora-app",
+    container_location=f"ghcr.io/sage-bionetworks/agora-app:{agora_version}",
+    container_port=4200,
+    container_memory=200,
+    container_env_vars={
+        "API_DOCS_URL": f"http://{fully_qualified_domain_name}/api-docs",
+        "APP_VERSION": f"{agora_version}",
+        "CSR_API_URL": f"http://{fully_qualified_domain_name}/api/v1",
```

Comment:
This is how it's configured in the [OpenChallenges deployment](https://github.com/Sage-Bionetworks-IT/openchallenges/blob/dev/app.py#L298-L305).  @tschaffter set it up that way so that's a good question for him.

Reply:
Yes, they should work if they are taken from the monorepo. Caddy is used as reverse proxy to make different paths available behind the same FQDN.